qiskit/aqua/operators/list_ops/summed_op.py

```diff
@@ -132,6 +134,29 @@ def reduce(self) -> OperatorBase:
         else:
             return cast(OperatorBase, reduced_ops)
 
+    def to_circuit(self) -> QuantumCircuit:
+        """Returns the quantum circuit, representing the SummedOp. In the first step,
+        the SummedOp is converted to MatrixOp. This is straightforward for most operators,
```

Comment:
The first line should be a one-line summary, then you can write the rest in a following paragraph.
```suggestion
        """Returns the quantum circuit, representing the SummedOp. 
        
        In the first step, the SummedOp is converted to MatrixOp. 
        This is straightforward for most operators,
```